### ISSUE TITLE
Incorrect axis order interpretation for urn:ogc:def:crs:epsg::4326 in WFS Janus

### ISSUE THREAD

**Issue by robinhoutmeyers** [opengeospatial/geoedge-plugfest#15]
The following XML POST request should return INLAND_WATERBODY_S features from the WFS source Janus available at https://externaltest.dev.geocloud.com/server/services/PuertoRicoVector/MapServer/WFSServer :
`
<?xml version='1.0' encoding='UTF-8'?>
<wfs:GetFeature xmlns:wfs="http://www.opengis.net/wfs/2.0" xmlns:fes="http://www.opengis.net/fes/2.0" xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:gml="http://www.opengis.net/gml/3.2" count="100" service="WFS" version="2.0.0">
  <wfs:Query xmlns:ns0="http://www.esri.com" typeNames="ns0:INLAND_WATERBODY_S">
    <fes:Filter>
      <fes:BBOX>
        <fes:ValueReference>ns0:SHAPE</fes:ValueReference>
        <gml:Envelope srsName="urn:ogc:def:crs:EPSG::4326">
          <gml:lowerCorner>17.578125 -67.5</gml:lowerCorner>
          <gml:upperCorner>18.28125 -66.09375</gml:upperCorner>
        </gml:Envelope>
      </fes:BBOX>
    </fes:Filter>
  </wfs:Query>
</wfs:GetFeature>
`
Following the axis order for urn:ogc:def:crs:EPSG::4326, lat lon is used. However, no features are returned.

Switching the lat lon axis order to lon lat does work, but it is expected that this should be the other way around - e.g. according to https://portal.opengeospatial.org/files/?artifact_id=24045.

**Comment by bermud** [comment]
Issue also reported by Client J "we had to manually specify axis order 1,2,3 for the features to render correctly."

**Comment by robinhoutmeyers** [comment]
Still present in sprint 2.

**Comment by bermud** [comment]
Client E report that he had to check **Invert Axis Orientation**  in QGIS in order for the queries to work.

**Comment by robinhoutmeyers** [comment]
Yes, but that is a workaround - which I assumed we should not use in an interoperability test.
